docusaurus.config.js

```diff
@@ -14,7 +14,7 @@ const docCollectionsLocation = './src/data/doc-collections';
 let cachedSources;
 
 const mixpanelOnLoad = `
-if ('${process.env.MIXPANEL_PROJECT_TOKEN}') {
+if (${process.env.MIXPANEL_PROJECT_TOKEN}) {
```

Comment:
this could be evaluated incorrectly
js on the page will treat this value as a variable name

Reply:
Undefined or 'undefined' I saw locally 'undefined' is truethy so I removed the quotes.

Reply:
```suggestion
if ('${process.env.MIXPANEL_PROJECT_TOKEN}' && '${process.env.MIXPANEL_PROJECT_TOKEN}' !== 'undefined') {
```